rest-api/tools/auth_setup.sh

```diff
@@ -88,10 +88,24 @@ function get_instance_connection_name {
 }
 
 function get_db_password {
+  user=$1
   echo "Getting database password..."
   tools/install_config.sh --key db_config --instance $INSTANCE \
       --creds_file ${CREDS_FILE} --config_output $TMP_DB_INFO_FILE
-  PASSWORD=`grep db_password $TMP_DB_INFO_FILE | cut -d\" -f4`
+  if [ "${user}" == "${RDR_DB_USER}" ]
```

Comment:
I think it'd be a bit cleaner to set a variable to db_password, read_only_db_password, or root_db_password in here, and have the actual PASSWORD=`...` command happen outside the if.